reactapplication/src/card.jsx

```diff
@@ -10,11 +15,16 @@ export default function Card({ testimonial, cardIndex, index }) {
         <img className="CardImage" {...testimonial.image} />
         <p className="reviewer">{testimonial.name}</p>
         <div className="">
+          <div className="rating">
+            {[...Array(rating)].map((e, i) => {
+              return <img className="ratingImage" {...ratingImage} />;
+            })}
+            {/* <img className="ratingImage" src="./images/star.png" alt="star" />
           <img className="ratingImage" src="./images/star.png" alt="star" />
           <img className="ratingImage" src="./images/star.png" alt="star" />
           <img className="ratingImage" src="./images/star.png" alt="star" />
-          <img className="ratingImage" src="./images/star.png" alt="star" />
```

Comment:
Code haru comment out garera rakhna bhanda ni baru completely remove gardida huncha
Purano code jaile ni git ko history bata revisit garna milcha

Of course, unless necessary for running differently.